packages/rest/src/lib/CDN.ts

```diff
@@ -108,8 +117,18 @@ export class CDN {
 		userId: string,
 		userAvatarDecoration: string,
 		options?: Readonly<BaseImageURLOptions>,
+	): string;
+
+	public avatarDecoration(
+		userIdOrAsset: string,
+		assetOrOptions?: Readonly<ImageURLOptions> | string,
+		options?: Readonly<BaseImageURLOptions>,
```

Comment:
Is there a reason you used `BaseImageURLOptions` in the implementation but `ImageURLOptions` in the overload?

Reply:
The new overload uses the `dynamicMakeURL()` method, which takes `ImageURLOptions`

Reply:
Both overloads use `ImageURLOptions`, this is the only place `BaseImageURLOptions` is used